src/layouts/learn-more.svelte

```diff
@@ -0,0 +1,125 @@
+<script context="module">
```

Comment:
Other pages does not seems to need `context="module"` attribute. Does it have special meaning?

Reply:
It allows module exports from the component